service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/PlacementController.java

```diff
@@ -133,6 +135,15 @@ private void sendNotificationForNewPlacementOrder(CallbackRequest callbackReques
         if (!isUpdatingExistingPlacement(previousPlacement, currentPlacement)) {
             publishPlacementApplicationUploadEvent(callbackRequest);
         }
+
+        var placement = mapper.convertValue(caseDetails.getData().get("placement"), Placement.class);
+        coreCaseDataService.triggerEvent(
+            callbackRequest.getCaseDetails().getJurisdiction(),
+            callbackRequest.getCaseDetails().getCaseTypeId(),
+            callbackRequest.getCaseDetails().getId(),
+            "internal-change:SEND_DOCUMENT",
+            Map.of("documentToBeSent", placement.getApplication())
+        );
```

Comment:
dont really fint into this method, there is already placement object so no need to get it again. What is more looks that placement can be removed in the end of about-to-submit as here we can get git using childId. Codes should check if document changed before sending